libs/nucs/src/validator.rs

```diff
@@ -205,7 +207,7 @@ impl NucValidator {
         Self::validate_condition(previous.audience == current.issuer, ValidationKind::IssuerAudienceMismatch)?;
         Self::validate_condition(previous.subject == current.subject, ValidationKind::DifferentSubjects)?;
         Self::validate_condition(
-            current.command.is_attenuation_of(&previous.command),
+            current.command.is_attenuation_of(&previous.command) || current.command.0 == REVOCATION_COMMAND,
```

Comment:
just checking, is any command  `nuc/revoke` is acceptable, or only `/nil/revoke`? If latter then we need to check `previous.command.starts_with(&["nil"])`

Reply:
Not sure I understand what you're saying, are you mixing up "nuc" with "nil"? There's no "nil/revoke" here. We only allow jumping to a command that's not an attenuation if you're jumping exactly to `/nuc/revoke`, nothing else, doesn't matter what the previous was.